UE/Application/Ports/IUserPort.hpp

```diff
@@ -18,6 +18,10 @@ class IUserPort
     virtual void showNotConnected() = 0;
     virtual void showConnecting() = 0;
     virtual void showConnected() = 0;
+
```

Comment:
delete blank line

Reply:
This blank line is OK for me, it visibly divides methods used for connecting UE from methods used for SMS